utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile

```
@@ -0,0 +1,8 @@
+
+FROM alpine/xml
+WORKDIR /client
+
+COPY ./utils/build/docker/dotnet/parametric/ApmTestClient.csproj .
+COPY ./utils/build/docker/dotnet/parametric/ddtracer_version.sh .
+RUN sh ddtracer_version.sh
```

As the sh file is mostly a one liner, you can shave it : 

```suggestion
RUN echo $(xmllint --xpath 'string(//Project/ItemGroup/PackageReference[@Include="Datadog.Trace.Bundle"]/@Version)' ApmTestClient.csproj) > SYSTEM_TESTS_LIBRARY_VERSION

```